teamcity: failed tests on master: acceptance/TestDockerCLI

The following tests appear to have failed:

[#857767](https://teamcity.cockroachdb.com/viewLog.html?buildId=857767):


```
--- FAIL: acceptance/TestDockerCLI (220.720s)
test_log_scope.go:81: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestDockerCLI146884331
test_log_scope.go:62: use -show-logs to present logs inline
--- FAIL: acceptance/TestDockerCLI: TestDockerCLI/test_exec_log.tcl (4.550s)

--- FAIL: acceptance/TestDockerCLI (220.720s)
test_log_scope.go:81: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestDockerCLI146884331
test_log_scope.go:62: use -show-logs to present logs inline
--- FAIL: acceptance/TestDockerCLI: TestDockerCLI/test_exec_log.tcl (4.550s)


```

Please assign, take a look and update the issue accordingly.

--- a/pkg/cli/interactive_tests/test_exec_log.tcl
+++ b/pkg/cli/interactive_tests/test_exec_log.tcl
@@ -14,13 +14,15 @@ system "if test -e $logfile; then false; fi"
 end_test
 
 start_test "Check that the exec log is created after enabled"
-send "SET CLUSTER SETTING sql.trace.log_statement_execute = TRUE;\r"
+send "SET CLUSTER   SETTING sql.trace.log_statement_execute = TRUE;\r"
+eexpect "SET CLUSTER SETTING"
 eexpect root@
 system "test -e $logfile"
 end_test
 
 start_test "Check that statements get logged to the exec log but stop logging when disabled"
-send "SELECT 'helloworld';\r"
+send "SELECT 'hello' || 'world';\r"
+eexpect "helloworld"
 eexpect root@
 
 # Errors must be logged too
@@ -33,13 +35,14 @@ eexpect root@
 # Check logging after disable
 send "SET CLUSTER SETTING sql.trace.log_statement_execute = FALSE;\r"
 eexpect root@
-send "SELECT 'lovely';\r"
+send "SELECT 'lov' || 'ely';\r"
+eexpect "lovely"
 eexpect root@
 
 flush_server_logs
 
 # Now check the items are there in the log file.
-system "grep -q helloworld $logfile"
+system "grep -q 'hello.*world' $logfile"
 system "grep -q nonexistent $logfile"
 system "if grep -q lovely $logfile; then false; fi"
 

